pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/export/json/structured/MatrixQuestionRecord.java

```diff
@@ -17,7 +17,7 @@ public final class MatrixQuestionRecord extends QuestionRecord {
     @SerializedName("matrixSelected")
     private Map<String, String> selected = new HashMap<>();
     @SerializedName("answer")
-    private Map<String, String> answer = new HashMap<>();
+    private List<String> answer = new ArrayList<>();
```

Comment:
Mapping of answer is expected to be text rather than object, hence failing .
To minimize impact on all existing answers, converted the matrix answer.
DSM is using "matrixSelected" : {} element in the matrix question. 
DSM is able to flag correct options checked/answered .